app/src/main/res/values/strings.xml

```diff
@@ -179,8 +179,14 @@
     <string name="no_recent_phrases_title">No recently used phrases</string>
     <string name="no_recent_phrases_message">Start using the app to see your most recently used phrases here.</string>
     <string name="categories_edit_title">Categories</string>
-    <!-- TODO: Remove or change this placeholder text -->
-    <string name="hello_blank_fragment">Hello blank fragment</string>
+
+    <!-- Camera Permissions -->
+    <string name="permissions_missing_dialog_title">Required Permissions Unavailable</string>
+    <string name="permissions_missing_dialog_body">Camera permissions are required to use the Head Tracking feature. Please enable them in the app settings.</string>
+    <string name="permissions_confirm">OK</string>
+    <string name="permissions_cancel">Cancel</string>
```

Comment:
Looks like we already have "Cancel" in `settings_dialog_cancel`. Wondering if we just use the pre-existing "Continue" as well instead of adding "OK". Less translation-changes to deal with.